ethereum/src/logs_handler.rs

```diff
@@ -0,0 +1,21 @@
+use crate::{
+	Result,
+	EthereumLikeChainClient
+};
+use async_trait::async_trait;
+use web3::types::{Log, H160, H256};
+
+#[async_trait]
+pub trait LogsHandler {
+	async fn handle(&self, client: &EthereumLikeChainClient, topics_list: &Vec<(H160, Vec<H256>)>, logs: Vec<Log>) -> Result<()>;
```

Comment:
Rename `topics_list` to `topics`, just like `logs` did. We haven't use logs_list right?